### PR TITLE
Security-only patch description for  release notes (10/8/2019)

### PR DESCRIPTION
**Pull request by jfrontain**
This pull request (PR) adds information about security patch 2.3.2.2 to the relevant release-related pages.

## Affected DevDocs pages
https://devdocs.magento.com/guides/v2.3/release-notes/bk-release-notes.html (2.3.x landing page)

https://devdocs.magento.com/guides/v2.3/release-notes/release-notes-2-3-3-commerce.html
https://devdocs.magento.com/guides/v2.3/release-notes/release-notes-2-3-3-open-source.html
https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.2OpenSource.html
https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.html

-->

### CODE REVIEW

File: guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md [hunk 1]
```diff
@@ -3,14 +3,22 @@ group: release-notes
 title: Magento Commerce 2.3.2 Release Notes
 ---
 
-*Patch code and release notes published on June 25, 2019. Release notes last updated on July 8, 2019.*
+*Patch code and release notes published on June 25, 2019. Release notes last updated on October 8, 2019.*
```
Comment (jeff-matthews) [inline guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md]:
```suggestion
*Patch code and release notes published on June 25, 2019.*
```

Comment (jeff-matthews) [reply guides/v2.3/release-notes/ReleaseNotes2.3.2Commerce.md]: We have autogenerated last modified time stamps on each topic now.

**Comment by jfrontain** [comment]
running tests

**Comment by jfrontain** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**